cmake/modules/RootMacros.cmake

```diff
@@ -16,6 +16,7 @@ if(WIN32)
   set(runtimedir ${CMAKE_INSTALL_BINDIR})
 elseif(APPLE)
   set(ld_library_path DYLD_LIBRARY_PATH)
+  set(ld_preload DYLD_INSERT_LIBRARIES)
```

Comment:
Sure this works on Catalina? https://theevilbit.github.io/posts/dyld_insert_libraries_dylib_injection_in_macos_osx_deep_dive/

Reply:
Absolutely not sure! So far, I was only able to test on ubuntu with clang. Therefore the PR ...

Reply:
However, that *would* be the correct variable. If you set up your OS X such that you can LD_PRELOAD, that's what you would have to add.

Reply:
Can we try to check somewhere in our infrastructure?

Reply:
Do you know how to disable source fortification or the other protection things on Mac? If somebody has this disabled, this variable *should* do the right thing. I thought it's better to provide something that *should* work than not to provide anything at all. Realistically, this won't work in our infrastructure for the foreseeable future, though.